packages/core/modules/enclosure_control.py

```diff
@@ -235,19 +236,18 @@ def auto_set_power_spectrometer(self) -> None:
         spectrometer in the morning when minimum angle is satisfied.
         """
 
-        current_sun_elevation = utils.Astronomy.get_current_sun_elevation()
-        min_power_elevation = (
-            self.config["general"]["min_sun_elevation"] - 1
-        ) * utils.Astronomy.units.deg
-
-        if current_sun_elevation is not None:
```

Comment:
Why could you remove this line?

Reply:
There is also a new test that draw random datetimes from each year between 2020 and 2050 and checks if the Astronomy class can generate an elevation for that: https://github.com/tum-esm/pyra/blob/integration-4.0.8/tests/utils/test_astronomy.py